easybuild/easyblocks/n/nwchem.py

```diff
@@ -405,6 +405,14 @@ def cleanup_step(self):
 
         super(EB_NWChem, self).cleanup_step()
 
+    def test_step(self):
+        """
+        Simply test that we can execute the binary - this fails sometimes with PMI errors
```

Comment:
Why does it fail sometimes? This test should always succeed if the binary is correctly built. Otherwise the test is wrong.